JitsiConference.ts

```diff
@@ -1257,12 +1384,12 @@ export default class JitsiConference {
    * Clear JitsiLocalTrack properties and listeners.
    * @param {JitsiLocalTrack} track - The JitsiLocalTrack object.
    */
-    onLocalTrackRemoved(track) {
+    onLocalTrackRemoved(track: JitsiLocalTrack): void {
         track.setConference(null);
         this.rtc.removeLocalTrack(track);
-        track.removeEventListener(JitsiTrackEvents.TRACK_MUTE_CHANGED, track.muteHandler);
+        track.removeEventListener(JitsiTrackEvents.TRACK_MUTE_CHANGED, this._fireMuteChangeEvent.bind(this, track));
```

Comment:
This doesn't work because `this._fireMuteChangeEvent.bind(this, track)` will return a new reference, which doesn't match the one you added. You need to save it, that's why it was assigned to `track.muteHandler`.

Reply:
When addint, you can use `addCancellableListener` which returns the remove function, so alll you need to do it call it.